packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -36,202 +55,113 @@ const indexActionsCellStyles = css({
   minWidth: spacing[5],
 });
 
-const indexActionsCellContainerStyles = css({
-  display: 'flex',
-  justifyContent: 'flex-end',
-});
-
-const tableHeaderStyles = css({
-  borderWidth: 0,
-  borderBottomWidth: 3,
-  '> div': {
-    justifyContent: 'space-between',
-  },
+const tableHeadRowStyles = css({
+  position: 'sticky',
```

Comment:
Any reason we are not using [TableHead](https://www.mongodb.design/component/table/documentation/) `isSticky` instead?

Reply:
🤯 I totally thought I looked for a prop like this. Nice catch thanks!